website/docs/tutorials/07-testing/03-fuzzing/strategies.md

```diff
@@ -27,10 +27,8 @@ func test_integers{syscall_ptr : felt*, range_check_ptr}(a : felt, b : felt) {
 }
 ```
 
-:::note
-By default, Protostar applies the [`strategy.felts()`](#strategyfelts) strategy to all felt
-parameters.
-:::
+It is required to either provide a strategy using [`given`](../02-cheatcodes/given.md)
+or specific values for parameters using [`example`](../02-cheatcodes/example.md).
```

Comment:
I wonder if this sentence is necessary

Reply:
We can remove it

Reply:
[remove sentence](https://github.com/software-mansion/protostar/pull/995/commits/4179ef53bc46bfb439929a359de8cc2c3c7a3d76)